exchanges/deribit/deribit_test.go

```diff
@@ -0,0 +1,3046 @@
+	assert.NoError(t, err)
+}
+func TestWSSubmitSell(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	info, err := d.GetInstrument(context.Background(), btcPerpInstrument)
+	require.NoError(t, err)
+	_, err = d.WSSubmitSell(&OrderBuyAndSellParams{
+		Instrument: btcPerpInstrument, OrderType: "limit",
+		Label: "testOrder", TimeInForce: "",
+		Trigger: "", Advanced: "", Amount: info.ContractSize * 3,
+		Price: 500000, MaxShow: 0, TriggerPrice: 0, PostOnly: false,
+		RejectPostOnly: false, ReduceOnly: false, MMP: false})
+	assert.NoError(t, err)
+}
+
+func TestEditOrderByLabel(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.EditOrderByLabel(context.Background(), &OrderBuyAndSellParams{Label: "incorrectUserLabel", Instrument: btcPerpInstrument,
+		Advanced: "", Amount: 1, Price: 30000, TriggerPrice: 0, PostOnly: false, ReduceOnly: false, RejectPostOnly: false, MMP: false})
+	assert.NoError(t, err)
+}
+
+func TestWSEditOrderByLabel(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.WSEditOrderByLabel(&OrderBuyAndSellParams{Label: "incorrectUserLabel", Instrument: btcPerpInstrument,
+		Advanced: "", Amount: 1, Price: 30000, TriggerPrice: 0, PostOnly: false, ReduceOnly: false, RejectPostOnly: false, MMP: false})
+	assert.NoError(t, err)
+}
+
+const submitCancelResponseJSON = `{"triggered": false, "trigger": "index_price", "time_in_force": "good_til_cancelled", "trigger_price": 144.73, "reduce_only": false, "profit_loss": 0, "price": "market_price", "post_only": false, "order_type": "stop_market", "order_state": "untriggered", "order_id": "ETH-SLIS-12", "max_show": 5, "last_update_timestamp": 1550575961291, "label": "", "is_liquidation": false, "instrument_name": "ETH-PERPETUAL", "direction": "sell", "creation_timestamp": 1550575961291, "api": false, "amount": 5 }`
+
+func TestSubmitCancel(t *testing.T) {
+	t.Parallel()
+	var resp *PrivateCancelData
+	err := json.Unmarshal([]byte(submitCancelResponseJSON), &resp)
+	require.NoError(t, err)
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err = d.SubmitCancel(context.Background(), "incorrectID")
+	assert.NoError(t, err)
+}
+
+func TestWSSubmitCancel(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.WSSubmitCancel("incorrectID")
+	assert.NoError(t, err)
+}
+
+func TestSubmitCancelAll(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.SubmitCancelAll(context.Background(), false)
+	assert.NoError(t, err)
+}
+
+func TestWSSubmitCancelAll(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.WSSubmitCancelAll()
```

Comment:
I believe all the endpoints [mentioned here](https://github.com/thrasher-corp/gocryptotrader/pull/1082#discussion_r1583915250) and their WS equivalent should be upgraded with the `detailed` bool